app/models/network_port.rb

```diff
@@ -0,0 +1,17 @@
+class NetworkPort < ActiveRecord::Base
+  include NewWithTypeStiMixin
+  include ReportableMixin
+
+  belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::CloudManager"
+  belongs_to :cloud_tenant
+  belongs_to :cloud_network
+  belongs_to :cloud_subnets
```

Comment:
I'm chasing my tail on this a little ... 

Here's a `::NetworkPort` as a base class.  And, it belongs to a bunch of cloudy things.

However, above I see a a subclass `InfraManager::NetworkPort < ::NetworkPort`.  So, there's an infra manager network port that derives from something that's clearly very cloudy.

How is the `InfraManager::NetworkPort` intended to be used?  What data is going to be stored there?  Is `::NetworkPort` really just for cloud stuff?  Or, should it also be used for infra stuff, too?

I'm really just confused why something clearly so cloudy would be the parent class for something in the infra manager.